src/features/Classes/ClassDetailPage/index.jsx

```diff
@@ -53,37 +72,54 @@ const ClassDetailPage = () => {
   const handleBackButton = () => (previousPage ? history.push(`/${previousPage}`) : history.push('/classes'));
 
   return (
-    <Container size="xl" className="px-4 mt-3">
-      <div className="d-flex justify-content-between mb-3 flex-column flex-sm-row">
-        <div className="d-flex align-items-center mb-3">
-          <Button onClick={handleBackButton} className="mr-3 link back-arrow" variant="tertiary">
-            <i className="fa-solid fa-arrow-left" />
-          </Button>
-          <h3 className="h2 mb-0 course-title">Class details: {classNameDecoded}</h3>
-        </div>
-      </div>
-      <Table
-        isLoading={isLoading}
-        columns={COLUMNS}
-        count={students.count}
-        data={students.data}
-        emptyText="No students found."
-        rowClassName="justify-content-center my-4 my-3"
-        colProps={{ xs: 11, className: 'px-0' }}
-      />
-
-      {students.numPages > 1 && (
-        <Pagination
-          paginationLabel="paginationNavigation"
-          pageCount={students.numPages}
-          currentPage={currentPage}
-          onPageSelect={handlePagination}
-          variant="reduced"
-          className="mx-auto pagination-table"
-          size="small"
-        />
+    <>
+      {!classInfo && (
+      <Container size="xl" className="px-4 mt-3 page-content-container d-flex justify-content-center">
+        <p>You must be an instructor in this class to see the information.</p>
+      </Container>
+      )}
+      {classInfo && (
+        <Container size="xl" className="px-4 mt-3">
+          <div className="d-flex justify-content-between mb-4 flex-column flex-sm-row">
+            <div className="d-flex align-items-center mb-2">
+              <Button onClick={handleBackButton} className="mr-3 link back-arrow" variant="tertiary">
+                <i className="fa-solid fa-arrow-left" />
+              </Button>
+              <h3 className="h2 mb-0 course-title">Class details: {classNameDecoded}</h3>
+            </div>
+            <Button
+              as="a"
+              href={classLink}
+              className="text-decoration-none text-white button-view-class mr-3"
+            >
+              <i className="fa-solid fa-arrow-up-right-from-square mr-2 mb-1" />
+              View class
+            </Button>
```

Comment:
This button should be aligned with the end of the instructor card:

![image](https://github.com/user-attachments/assets/2f55154c-afe9-49dd-9fe2-b5b95a0b9641)